components/events/eventsHeader.tsx

```diff
@@ -1,14 +1,23 @@
-import Image from "next/image";
+import Image, { getImageProps } from "next/image";
+import { getBackgroundImage } from "../../helpers/images";
```

Comment:
maybe change this to make more meaningful name.

Reply:
i think this is probably ok - see https://nextjs.org/docs/pages/api-reference/components/image#background-css where I've taken the function from